### PR TITLE
UAR-168: Add Feature flag for ROE Update and Remove

### PR DESCRIPTION
**Pull request by jkayyala**
### JIRA link
[UAR-168](https://companieshouse.atlassian.net/browse/UAR-168)

### Change description

Adding feature flag for ROE Update and remove

### CODE REVIEW

**Comment by markpit** [comment]
looks ok to me

**Review by markpit — APPROVED** [review]

**Comment by markpit** [comment]
actually just spotted something with the naming so don't merge just yet

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java [hunk 1]
```diff
@@ -50,6 +50,12 @@ public class OverseasEntitiesController {
     @Value("${FEATURE_FLAG_ENABLE_VALIDATION_25082022}")
     private boolean isValidationEnabled;
 
+    @Value("${FEATURE_FLAG_ROE_UPDATE_24112022}")
```
Comment (markpit) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java:53]: could you include the word ENABLE in the feature flag names, it should make it clear that the true/false is to enable/disable and keeps it aligned to our current one.
so 
FEATURE_FLAG_ROE_UPDATE_24112022 -> FEATURE_FLAG_ENABLE_ROE_UPDATE_24112022
FEATURE_FLAG_ROE_REMOVE_24112022 -> FEATURE_FLAG_ENABLE_ROE_REMOVE_24112022

will need to update the docker PR as well to match the names

Comment (jkayyala) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java:53]: Sure, will update the PR's to add it in

**Review by markpit — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Review by AndreKramer7654 — APPROVED** [review]

**Review by AndreKramer7654 — APPROVED** [review]

**Pull request merged**